daml-assistant/daml-helper/src/DA/Daml/Helper/Start.hs

```diff
@@ -168,12 +168,13 @@ withJsonApi (SandboxPort sandboxPort) (JsonApiPort jsonApiPort) extraArgs a = do
             , "--http-port", show jsonApiPort
             , "--allow-insecure-tokens"
             ] ++ extraArgs
+    putStrLn ("Starting JSON API: " <> unwords args)
```

Comment:
did you intentionally leave this in? Seems a bit inconsistent to have this for the JSON API but not for other stuff, e.g., Navigator.

Reply:
Yep, oops!